src/components/proofs/ProofList.tsx

```diff
@@ -1,49 +1,36 @@
-import { AccentText, BodyText } from 'components/Text'
+import { AccentText } from 'components/ui/Text'
+import { DataKeys } from 'models/DataKeys'
 import { Suspense } from 'preact/compat'
 import { space } from 'classnames/tailwind'
 import { useSnapshot } from 'valtio'
 import ERC721ProofSection from 'components/proofs/ERC721ProofSection'
-import EmailProof from 'components/proofs/EmailProof'
-import ProofSection from 'components/ProofSection'
+import EmailProofSection from 'components/proofs/EmailProofSection'
 import ProofStore from 'stores/ProofStore'
-import ReadyEmailProof from 'components/proofs/ReadyEmailProof'
-import Scrollbar from 'components/Scrollbar'
-import WalletStore from 'stores/WalletStore'
+import Scrollbar from 'components/ui/Scrollbar'
 import data, { BadgeSourceType } from 'data'
 
 export function ProofListSuspended() {
-  const { account } = useSnapshot(WalletStore)
-  const { emailProofsCompleted, proofsCompleted } = useSnapshot(ProofStore)
+  const stores = useSnapshot(ProofStore)
+
+  const hasCompletedProofs = Object.values(stores).some(
```

Comment:
You're doing this twice, maybe it's better to extract it into `helper` or put inside `ProofStore`?